Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,30 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+ let durationInHours = tasks.map(hours => hours.duration / 60 ); 
```

Comment:
I like that you created individual variables which hold different states of information instead of having a single variable trying to hold all values at different stages of time. However, we don't need to use `let` keyword here to create those variables. Please try to use `const` here for all variable initialisations.

The reason being that `map`, `filter` and `reduce` methods create a new array everytime and hence the variables `durationInHours`, `moreThanTwo` and others don't change their values. 🙂